ui/lib/core/addon/components/page-header.js

```diff
@@ -3,11 +3,8 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
-import Component from '@ember/component';
-import layout from '../templates/components/page-header';
+import Component from '@glimmer/component';
 
-export default Component.extend({
-  layout,
-  tagName: '',
-  hasLevel: true,
-});
+export default class PageHeader extends Component {
+  hasLevel = true;
```

Comment:
If this is intended to be overwritable and not a static value, we should instead use:
```suggestion
get hasLevel() { return this.args.hasLevel === undefined ? true : this.args.hasLevel; }
```
You could also opt to explicitly check if the type of hasLevel is boolean and if not default to our default value `true`